commands/bench/lib/bench-all-substrate.sh

```diff
@@ -53,6 +53,10 @@ EXCLUDED_PALLETS=(
   "pallet_offences"
   # Only used for testing, does not need real weights.
   "frame_benchmarking_pallet_pov"
```

Comment:
Really? We have the excluded list here in this repo? @paritytech/opstooling please fix this and move the excluded list to the repo where this is executed, aka `polkadot-sdk`.

Reply:
Could you provide more context on what happened and how would you see the fix implemented?

Reply:
yea it was copied from substate repo while ago as is. Will move it to polkadot-sdk then

Reply:
@bkchr we're thinking to actually move all the commands to polkadot-sdk when we migrate to cmd bot as GHA. @Bullrich is on it. For now I will keep it here though, or if you think it won't wait few weeks - we can move the definition of excluded list of pallets